client/views/meet/MeetPage.js

```diff
@@ -0,0 +1,59 @@
+import { Meteor } from 'meteor/meteor';
+import React, { useEffect, useState, useCallback } from 'react';
+
+import { APIClient } from '../../../app/utils/client';
+import { useRouteParameter, useQueryStringParameter } from '../../contexts/RouterContext';
+import NotFoundPage from '../notFound/NotFoundPage';
+import PageLoading from '../root/PageLoading';
+import CallPage from './CallPage';
+
+function MeetPage() {
+	const [isRoomMember, setIsRoomMember] = useState(false);
+	const [status, setStatus] = useState(null);
+	const roomId = useRouteParameter('rid');
+	const visitorToken = useQueryStringParameter('token');
+	const visitorId = useQueryStringParameter('id');
```

Comment:
Passing ID in query param is redundant since we already have the token in the query param and we can get the id using token